otter/models/cass.py

```diff
@@ -1318,12 +1318,12 @@ def _delete_lock_znode(result):
                     exc=f.value,
                     otter_msg_type="ignore-delete-lock-error"))
 
-        lock = self.kz_client.Lock(LOCK_PATH + '/' + self.uuid)
-        lock.acquire = functools.partial(lock.acquire, timeout=120)
+        lock = zk.PollingLock(self.dispatcher, LOCK_PATH + '/' + self.uuid)
+        lock.acquire = functools.partial(lock.acquire, timeout=10)
```

Comment:
Magic number - also - why is it going down by an order of magnitude?

Does this really need to be 10 for specific correctness reasons or should it be configurable?